sunpy/map/mapbase.py

```diff
@@ -2765,6 +2767,78 @@ def contour(self, level, **kwargs):
         contours = [self.wcs.array_index_to_world(c[:, 0], c[:, 1]) for c in contours]
         return contours
 
+    def get_contours(self, level, method='contourpy', **kwargs):
+        """
+        Returns coordinates of the contours for a given level value.
+
+        For details of the contouring algorithm, see :func:`contourpy.contour_generator` or :func:`contourpy.contour_generator`.
+
+        Parameters
+        ----------
+        level : float, astropy.units.Quantity
+            Value along which to find contours in the array. If the map unit attribute
+            is not `None`, this must be a `~astropy.units.Quantity` with units
+            equivalent to the map data units.
+        method : {'contourpy', 'skimage'}
+            Determines which contouring method is used and should
+            be specified as either 'contourpy' or 'skimage'.
+            Defaults to 'contourpy'.
+        kwargs :
+            Additional keyword arguments passed to either :func:`contourpy.contour_generator`
+            or :func:`skimage.measure.find_contours`, depending on the value of the ``method`` argument.
+
+        Returns
+        -------
+        contours: list of (n,2) `~astropy.coordinates.SkyCoord`
+            Coordinates of each contour.
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> import sunpy.map
+        >>> import sunpy.data.sample  # doctest: +REMOTE_DATA
+        >>> aia = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)  # doctest: +REMOTE_DATA
+        >>> contours = aia.get_contours(50000 * u.DN, method='contourpy')  # doctest: +REMOTE_DATA
+        >>> print(contours[0])  # doctest: +IGNORE_WARNINGS
```

Comment:
This is the wrong place to make this change and it still needs the REMOTE_DATA

Reply:
Sorry, where can I add +IGNORE_WARNINGS then?

Reply:
It gets added to the example for the old method which is raising the deprecation warning.

Reply:
Sorry, yeah it should be for contour